Documentation/devicetree/bindings/media/i2c/arducam,64mp.yaml

```diff
@@ -0,0 +1,113 @@
+# SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+%YAML 1.2
+---
+$id: http://devicetree.org/schemas/media/i2c/arducam,64mp.yaml#
+$schema: http://devicetree.org/meta-schemas/core.yaml#
+
+title: Arducam 1/1.7-Inch 64Mpixel CMOS Digital Image Sensor
+
+maintainers:
+  - Lee Jackson <info@arducam.com>
+
+description: |-
+  The Arducam 1/1.7-Inch 64Mpixel CMOS active pixel digital image sensor
+  with an active array size of 9248 x 6944. It is programmable through
+  I2C interface. The I2C address is fixed to 0x1A as per sensor data sheet.
+  Image data is sent through MIPI CSI-2, which is configured as either 2 or
```

Comment:
which can be configured for operation with either 2 or 4 data lanes.

Reply:
done.